extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergCatalogAdapter.java

```diff
@@ -1211,16 +1168,13 @@ private static List<CompletedParquetWrite> writeParquet(
                 onWriteCompleted, icebergTable.schema().idToName());
 
         // Write the data to parquet files
+        int count = 0;
         for (final Table dhTable : dhTables) {
-            final long epochMicrosNow = Instant.now().getEpochSecond() * 1_000_000 + Instant.now().getNano() / 1_000;
-            final String filename = new StringBuilder()
-                    .append(epochMicrosNow) // To keep the data ordered by time
-                    .append("-")
-                    .append(UUID.randomUUID())
-                    .append(".parquet")
-                    .toString();
-            final String newDataLocation =
-                    icebergTable.locationProvider().newDataLocation(filename);
+            final String filename = String.format(
+                    "00000-%d-%s.parquet",
```

Comment:
This does not preserve the order if you have more than 10 tables

```
00000-0-xyz.parquet
00000-1-abc.parquet
00000-10-abc.parquet
00000-2-abc.parquet
...
```

and technically, filesystem ordering is not guaranteed regardless.

Reply:
I agree, its more like a best effort.
Also, its unlikely we will write more than 10 tables in a single write call.